lib/dictBuilder/fastcover.c

```diff
@@ -148,6 +147,9 @@ static COVER_segment_t FASTCOVER_selectSegment(const FASTCOVER_ctx_t *ctx,
   COVER_segment_t bestSegment = {0, 0, 0};
   COVER_segment_t activeSegment;
 
+  /* Initialize array to keep track of frequency of dmer within activeSegment */
+  U16* segmentFreqs = (U16 *)calloc(((U64)1 << f), sizeof(U16));
```

Comment:
This doesn't belong in selectSegment, we only need one copy per thread, but we don't need to allocate/free it in the hot loop. This probably belongs in `FASTCOVER_buildDictionary()`.